cirq/sim/mux.py

```diff
@@ -22,11 +22,21 @@
 import numpy as np
 
 from cirq import circuits, protocols, study, schedules, devices, ops, value
-from cirq.sim import sparse_simulator, density_matrix_simulator
+from cirq._doc import document
+from cirq.sim import (sparse_simulator, density_matrix_simulator,
+                      wave_function_simulator)
 
 if TYPE_CHECKING:
     import cirq
 
+CIRCUIT_LIKE = Union[circuits.Circuit, ops.Gate, ops.OP_TREE, schedules.
+                     Schedule]
+document(
+    CIRCUIT_LIKE,  # type: ignore
+    """A `cirq.Circuit`, a `schedules.Schedule`,  or a value that can be
+    trivially converted into one of these.
```

Comment:
Explain the values that can be trivially converted: a gate, an operation, and a list or tree of operations.